src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -47,123 +52,51 @@ const queryClient = new QueryClient({
-
-  useEffect(() => {
-    if (bundle?.path?.path) {
-      setEntityName(bundle.path.path[bundle.path.path.length - 1].name)
-      const path = bundle.path.path.slice(1, bundle.path.path.length - 1) // drop the first element, which is always syn4489 "root"
-      const _fullPath = path.map(header => header.name).join('/')
-      setFullPath(_fullPath)
-      if (path.length < 4) {
-        // Show the full path from project to entity
-        setDisplayedPath(_fullPath)
-      } else {
-        // Truncate the path, showing only project, parent, and self
-        setDisplayedPath(
-          path[0].name + // Project
-            '/…/' +
-            path
-              .slice(path.length - 1) // drop everything except parent and self
-              .map(header => header.name)
-              .join('/'),
-        )
-      }
-    }
-  }, [bundle])
-
-  return (
-    <div className="EntityFinder__Selected__Row">
-      <ReactTooltip id={ENTITY_PATH_TOOLTIP_ID} delayShow={500} place={'top'} />
-      <span
-        className="EntityFinder__Selected__Row__DeselectButton"
-        onClick={() => {
-          toggleSelection(entity)
-        }}
-      >
-        <span className="fa-layers fa-fw">
-          <FontAwesomeIcon
-            className="EntityFinder__Selected__Row__DeselectButton__IconCircle"
-            icon={faCircle}
-            size="lg"
-          />
-          <FontAwesomeIcon
-            className="EntityFinder__Selected__Row__DeselectButton__IconCross"
-            size={'sm'}
-            icon={faTimes}
-          />
-        </span>
-      </span>
-      <span
-        data-for={ENTITY_PATH_TOOLTIP_ID}
-        data-tip={`${fullPath}/${entityName}`}
-      >
-        {displayedPath ? displayedPath + '/' : ''}
-      </span>
-      <span className="EntityFinder__Selected__DeselectButton__EntityName">
-        {entityName}
-      </span>
-      {entity.targetVersionNumber && (
-        <span> (Version {entity.targetVersionNumber})</span>
-      )}
-    </div>
-  )
-}
-
 export type EntityFinderProps = {
   sessionToken: string
-  /** Required if `initialScope` is The SynID of the entity that should open by default. This dictates the 'Current Project' */
-  initialContainerId: string
   /** Whether or not it is possible to select multiple entities */
   selectMultiple: boolean
   /** Callback invoked when the selection changes */
   onSelectedChange: (selected: Reference[]) => void
   /** The initial appearance of the entity finder. Possible values include "Current Project", "All Projects", "Projects Created By Me", "My Favorites" */
   initialScope: FinderScope
+  /** The SynID of the 'Current Project'. If this is not a defined, then the scope cannot be "Current Project" */
+  projectId?: string
+  /** The SynID of the entity that should open by default. This dictates the 'Current Project'. If this is not a synID, then the scope cannot be "Current Project" */
+  initialContainer: string | 'root' | null
```

Comment:
Decoupled the project ID (used to determine the "Current Project") from the initial container (which may not be a Synapse entity, e.g. if you wanted the root node in the tree to be initially selected)